sos/report/plugins/smartcard.py

```diff
@@ -19,14 +19,17 @@ class Smartcard(Plugin, RedHatPlugin):
     profiles = ('security', 'identity', 'hardware')
 
     files = ('/etc/pam_pkcs11/pam_pkcs11.conf',)
-    packages = ('pam_pkcs11', 'pcsc-tools', 'opensc')
+    # The pam_pkcs11 is available only in RHEL7
+    packages = ('pam_pkcs11', 'pcsc-tools', 'opensc', 'pcsc-lite', 'pcsc-lite-ccid')
```

Comment:
There is a PEP error / too long line:

    sos/report/plugins/smartcard.py:23:80: E501 line too long (84 > 79 characters)

Please split the line like e.g. https://github.com/sosreport/sos/blob/main/sos/report/plugins/openvswitch.py#L365-L367 and verify via e.g. `pycodestyle sos/report/plugins/smartcard.py`.

Reply:
I was trying to set up the git commit hooks according to the guide, but it is not updated for 5 years and no longer reasonably working -- can you update the scripts to use the pycodestyle instead of the pep8 at least?

https://github.com/sosreport/sos/wiki/Using-git-hooks-to-automate-tests

Reply:
Fixed with current version. The tests did not work for me locally on Fedora so I gave up and hope they will run here somehow.